frontend/templates/WelcomeScreen/WelcomeScreen.jsx

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable */
+import { useRef, useState } from 'react';
+
+import { Grid, Typography } from '@mui/material';
+
+import GradientOutlinedButton from '@/components/GradientOutlinedButton';
+
+import ArrowDropDown from '@/assets/svg/DropDown.svg';
+
+import ExplainCom from './ProcessbarExplain/index';
+import StatusIcon from './StatusIcon/index';
+
+import styles from './styles';
+
+import theme from '@/theme/theme';
+
+const WelcomeScreen = (props) => {
+  const guidList = useRef([
+    {
+      type: 'Dot',
+      status: 'doing',
+    },
+    {
+      type: 'Line',
+      status: 'undo',
+    },
+    {
+      type: 'Dot',
+    },
+    {
+      type: 'Line',
+    },
+    {
+      type: 'Dot',
+    },
+    {
+      type: 'Line',
+    },
+    {
+      type: 'Dot',
+    },
+  ]);
+  const [explainShow, setExplainShow] = useState(false);

```

Comment:
Please make sure you follow a naming convention which is easy for others to understand. I don't understand what this variable explainShow is for.